cmd/mattermost/commands/command_test.go

```diff
@@ -169,3 +169,53 @@ func TestDeleteCommand(t *testing.T) {
 		assert.Error(t, th.RunCommand(t, "command", "delete", "invalid"))
 	})
 }
+
+func TestModifyCommand(t *testing.T)  {
+	th := Setup().InitBasic()
+	defer th.TearDown()
+	team := th.BasicTeam
+	user := th.BasicUser
+	th.LinkUserToTeam(user, team)
+
+	originalCommand := &model.Command{
+		DisplayName: "dn_" + model.NewId(),
+		Method:      "G",
+		TeamId:      team.Id,
+		Username:    user.Username,
+		CreatorId:   user.Id,
+		Trigger:     "trigger_" + model.NewId(),
+		Description :"old description",
+		URL : "http://localhost:8000/modify-command",
+		IconURL : "original icon",
+		AutoComplete : false,
+	}
+	modifiedTitle := "modified title"
+	modifiedDescription := "modified description"
+	modifiedTrigger := "modified trigger"
```

Comment:
This doesn't match with the assertion at the end.

Reply:
And this trigger is invalid because the "no-spaces-on-triggers" check.

Reply:
Updated the test values for triggers and others